packages/core/src/checkout/checkout-service.spec.ts

```diff
@@ -200,19 +199,19 @@ describe('CheckoutService', () => {
             getResponse(getCountriesResponseBody()),
         );
 
-        paymentStrategy = new OfflinePaymentStrategy(store, orderActionCreator);
+        paymentStrategyRegistry = new PaymentStrategyRegistry(store);
```

Comment:
Do we need to change this?

Reply:
This looks bit confusing, it is the same line as the line213 before.
I put `const paymentIntegrationService` up first, then `paymentStrategy` and `jest.spyOn`. Somehow, this line got moving up.

Reply:
I moved it next to the `paymentStrategyRegistryV2` now.